src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/ReflectionMemberAccessor.cs

```diff
@@ -38,7 +38,7 @@ public ConstructorContext([DynamicallyAccessedMembers(DynamicallyAccessedMemberT
                 return null;
             }
 
-            return new ConstructorContext(type).CreateInstance;
+            return new ConstructorContext(type).CreateInstance!;
```

Comment:
Bang seems unnecessary?

Reply:
`CreateInstance` returns nullable because it accounts for `Nullable<T>` but we cannot return null here (it's only used by JsonTypeInfoKind.Object and we treat `Nullable` specially)